### PR TITLE
Production HSES import fix

### PR DESCRIPTION
**Pull request by rahearn**
**Description of change**

* Handle `nil` grant start and end dates
* Add ability to kick off an import via CLI

**How to test**

Production data imports successfully. Ryan tested this on GFE with a production dataset.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/380

**Checklist**

- [x] Meets issue criteria
- [x] Code tested
- [x] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

### CODE REVIEW

**Comment by rahearn** [comment]
This has been pushed to production as a hotfix.

File: src/lib/updateGrantsGrantees.js [hunk 1]
```diff
@@ -56,15 +53,20 @@ export async function processFiles() {
...
-      number: g.grant_number,
-      regionId: parseInt(g.region_id, 10),
-      granteeId: parseInt(g.agency_id, 10),
-      status: g.grant_status,
-      startDate: g.grant_start_date,
-      endDate: g.grant_end_date,
-    }));
+    const grantsForDb = grant.grant_awards.grant_award.map((g) => {
+      let { grant_start_date: startDate, grant_end_date: endDate } = g;
+      if (typeof startDate === 'object') { startDate = null; }
```
Comment (rahearn) [inline src/lib/updateGrantsGrantees.js:58]: nil dates in the xml end up being `{ "xsi:nil": true }` here

**Review by dcmcand — APPROVED** [review]

**Comment by kryswisnaskas** [comment]
> Ryan tested this on GFE with a production dataset.
Fantastic. Assuming this was tested over the HSES grantee data pull?

**Review by jasalisbury — APPROVED** [review]

**Comment by rahearn** [comment]
> Assuming this was tested over the HSES grantee data pull?

correct

**Pull request merged**